exec/java-exec/src/main/java/org/apache/drill/exec/planner/fragment/Wrapper.java

```diff
@@ -225,12 +236,12 @@ public void computeCpuResources() {
       return result;
     };
 
-    Function<DrillbitEndpoint, NodeResource> cpuPerEndpoint = (endpoint) -> new NodeResource(1, 0);
+    Function<DrillbitEndpoint, NodeResource> cpuPerEndpoint = (
+        endpoint) -> new NodeResource(1, 0);
```

Comment:
Please remove brackets:
```suggestion
    Function<DrillbitEndpoint, NodeResource> cpuPerEndpoint =
        endpoint -> new NodeResource(1, 0);
```